### PR TITLE
Django4 ubuntu22

### PR DESCRIPTION
**Pull request by baloola**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -93,9 +93,9 @@ def fullsplit(path, result=None):
         "tools/eoxserver-preprocess.py"
     ],
     install_requires=[
-        'django<3',
+        'django<5',
```
Comment (constantinius) [inline setup.py:96]: We use Django 3.2 in the tests, So I guess we should use `django<4` here

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
-gunicorn
-eventlet==0.30.2
-greenlet>=0.4.14
\ No newline at end of file
+https://github.com/benoitc/gunicorn/archive/refs/heads/master.zip#egg=gunicorn==20.1.0
+https://github.com/eventlet/eventlet/archive/master.zip
```
Comment (constantinius) [inline requirements.txt:2]: I think we can remove greenlet/eventlet from the requirements if they are not used

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+greenlet>=0.4.14
+django-utils-six==2.0
```
Comment (constantinius) [inline requirements.txt:4]: Is this actually needed anywhere?

File: setup.py (hunk 1 above)
Comment (baloola) [reply setup.py]: changed to django<4

File: requirements.txt (hunk 2 above)
Comment (baloola) [reply requirements.txt]: used guincorn standerd package and removed greenlet/eventlet

File: requirements.txt (hunk 3 above)
Comment (baloola) [reply requirements.txt:2]: django-utils-six==2.0 is still needed currently

**Pull request merged**